agent/src/main/resources/application.yml

```diff
@@ -55,6 +55,8 @@ app:
       count-threshold: ${STATE_CHANGE_COUNT_THRESHOLD:12}
       window-time: ${STATE_CHANGE_WINDOW_TIME:5}
       recovery-time: ${STATE_RECOVERY_TIME:3}
+  storage:
+    type: AZURE # LOCAL, AZURE, ALICLOUD, etc.
```

Comment:
Why agent side also need to config this? Could the agent just follow the center on this?

Reply:
Removed